data/json/items/vehicle/solar.json

```diff
@@ -70,6 +71,7 @@
     "symbol": ";",
     "volume": "250 ml",
     "category": "spare_parts",
+    "longest_side": "25 cm",
```

Comment:
I understand you arrived at this value based on a 4x3 configuration of cells in the overall panel, but I don't think real-life cells are nearly this large. Something closer to 6x6 inches (~15x15 cm) is more typical of the cells in real-life panels. [Reference](https://unboundsolar.com/blog/solar-panel-size-guide)

Since the recipe only takes 12 of these, that would suggest the total panel size should be much smaller as well.

Reply:
Alright, so supposing its only a 60x45cm solar panel, that would roughly put its weight around 7 pounds with a metal frame and converter.

Reply:
Yeah that sounds reasonable.

Another piece of info you may have already considered is that the standard solar panel vehicle part generates 50 watts of power. This would seem to align well with having 12 cells in the panel, based on [this source](https://letsavelectricity.com/what-is-the-size-of-a-solar-panel-weight-of-a-solar-panel/) which says "The standard size of a 250 watt 60 cell solar panel is 39 inch x 66 inch" (167x100 cm ~= 16,700 cm^2) - so dimensions around one-fifth that area should be about right for 50 watts (60x45cm ~= 2700 cm^2, pretty close), and it's exactly one-fifth the number of cells.

The individual solar cell weight of 500g seems a bit steep for 15x15cm, so that could be reduced if it makes the math come out a little better.

Reply:
I've adjusted the values, reducing the panel itself to 60 cm and 3200 grams, while solar cells are down to 15 cm and 200 grams.  Given twelve cells are present, this gives a total weight of 2400g from the cells alone, leaving 800g to be taken up by the power converter and frame.

Reply:
Usually solar panels covered with glass, 60x45cm 3mm thick sheet of glass would weight about 2kg. But solar panel recipe don't contain glass, maybe oversight. 

As an option we could add glass to the recipe and adjust total mass of the solar panel. So it wasn't too lightweight, afterall glass in there irl for a reason.